frontend/pages/g/_groupSlug/r/create/url.vue

```diff
@@ -94,6 +95,11 @@ export default defineComponent({
     const router = useRouter();
     const tags = useTagStore();
 
+    const bulkImporterTarget = computed(() => {
+      const urlIndex = route.value.fullPath.lastIndexOf("/url");
+      return route.value.fullPath.slice(0, urlIndex) + "/bulk";
+    });
```

Comment:
I think you can simplify that and just use:

```suggestion
    const bulkImporterTarget = computed(() => `/g/${groupSlug.value}/r/create/bulk`);
```

Reply:
I see what you mean with won't update the menu, forget this

Reply:
I feel like there was a reason I did it this way, but I can't remember. Thanks!

Reply:
only reason i can think of is that is uses the route. 
So if somebody would change those (can't imagine why somebody would do that) you would not have to update the component.